src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 7) {
+    total = (days * 40) - 50;
+  } else if (days >= 3 && days < 7) {
+    total = (days * 40) - 20;
+  } else {
+    total = days * 40;
```

Comment:
Good job!
But lets make some variables for values, because for now we don't understand what it is - 40, 50, 20 ...